mlir/test/Conversion/EmulateFp8ExtTrunc/emulate-ocp-fp8-ext-trunc.mlir

```diff
@@ -0,0 +1,100 @@
+// RUN: rocmlir-opt -pass-pipeline='builtin.module(gpu.module(emulate-fp8-ext-trunc),emulate-fp8-ext-trunc)' -split-input-file %s | FileCheck %s
```

Comment:
Same intent as emulate-fp8-ext-trunc.mlir, but enough changes that I didn't want to merge with it.